pkg/app/piped/apistore/eventstore/store.go

```diff
@@ -116,27 +124,49 @@ func (s *store) sync(ctx context.Context) error {
 		return nil
```

Comment:
Should we clear the local list `notHandledEvents`?

Reply:
There you go! Well noticed.

Reply:
Addressed https://github.com/pipe-cd/pipecd/pull/3124/commits/6bcc77a1edbbd306ba1071a5f43df80f131ad83c